src/atoms/phoneInputField/PhoneInputField.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {
+  ChangeEventHandler,
+  InputHTMLAttributes,
+  MouseEventHandler,
+} from 'react'
+import classes from './phoneInputField.module.scss'
+import { QuestionLabel } from '../questionLabel'
+import 'react-international-phone/style.css';
+import {
+  CountrySelector,
+  usePhoneInput,
+} from 'react-international-phone';
+import { getDefaultCountries } from './helpers';
+import { useValidate } from '../../hooks/useValidate';
+
+export interface PhoneInputFieldProps extends InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * sets label of the button
+   */
+  label: string
+  /**
+   * sets id that is used to connect input with label
+   */
+  id: string
+  /**
+   * change event handler
+   */
+  onChange: ChangeEventHandler<HTMLInputElement>
+  /**
+   * click event handler
+   */
+  onClick?: MouseEventHandler<HTMLInputElement>
+  /**
+   * Is the question required?
+   */
+  mandatory?: boolean
+
+  /**
+   * The value of the input
+   */
+  value: string
+}
+
+export const PhoneInputField = ({
+  onChange,
+  id,
+  label,
+  type,
+  mandatory,
+  value,
+  ...props
+}: PhoneInputFieldProps): JSX.Element => {
+  const { isPossibleE164Number } = useValidate()
+  const { phone, handlePhoneValueChange, inputRef, country, setCountry } =
+    usePhoneInput({
+      initialCountry: 'us',
```

Comment:
I would make all parameters that influence the look & feel of the component (optional) parameters with a sensible default. That way the consuming app (eg: Hosted Pages) can determine what the values should be based on the locale.

```javascript
export const PhoneInputField = ({
  onChange,
  id,
  label,
  type,
  mandatory,
  value,
  initialCountry = 'us',
  hideSpaceAfterDialCode = true,
  ...props
}: PhoneInputFieldProps): JSX.Element => {
  const { isPossibleE164Number } = useValidate()
  const { phone, handlePhoneValueChange, inputRef, country, setCountry } =
    usePhoneInput({
      initialCountry,
      value: value,
      hideSpaceAfterDialCode,
      countries: getDefaultCountries(),
    });
```